printing/printer.ml

```diff
@@ -751,63 +754,38 @@ let default_pr_subgoals ?(pr_first=true)
     | Some cmd -> Feedback.msg_info cmd
     | None -> ()
   in
+
+  (** Main function *)
   match goals with
   | [] ->
       begin
-	let exl = Evd.undefined_map sigma in
-	if Evar.Map.is_empty exl then
-	  (str"No more subgoals." ++ print_dependent_evars None sigma seeds)
-	else
+        let exl = Evd.undefined_map sigma in
+        if Evar.Map.is_empty exl then
+          (str"No more subgoals." ++ print_dependent_evars None sigma seeds)
+        else
           let pei = pr_evars_int sigma ~shelf ~givenup:[] 1 exl in
-	  v 0 ((str "No more subgoals,"
-                ++ str " but there are non-instantiated existential variables:"
-	        ++ cut () ++ (hov 0 pei)
-                ++ print_dependent_evars None sigma seeds
-                ++ cut () ++ str "You can use Grab Existential Variables."))
+          v 0 ((str "No more subgoals,"
+                      ++ str " but there are non-instantiated existential variables:"
+                      ++ cut () ++ (hov 0 pei)
+                      ++ print_dependent_evars None sigma seeds
+                      ++ cut () ++ str "You can use Grab Existential Variables."))
       end
   | g1::rest ->
       let goals = print_multiple_goals g1 rest in
       let ngoals = List.length rest+1 in
       v 0 (
-	int ngoals ++ focused_if_needed ++ str(String.plural ngoals "subgoal")
+	      int ngoals ++ focused_if_needed ++ str(String.plural ngoals "subgoal")
```

Comment:
> This message is on a line I didn't change.

This line got reindented so it was touched.

`git rebase --whitespace=fix HEAD~4` should fix it.